app/src/auto_validator/core/admin.py

```diff
@@ -1,15 +1,19 @@
+from typing import Any
+
 from django.contrib import admin  # noqa
 from django.contrib.admin import register  # noqa
+from django.db.models import Case, IntegerField, Value, When  # noqa
```

Comment:
Why noqa? that doesn't seem like a great idea in general. Can be used if there is a really good reason. What's the reason?